src/filesystem/implementation.js

```diff
@@ -1633,6 +1663,7 @@ function validate_file_options(options, enc, fileMode){
 }
 
 function open(context, path, flags, mode, callback) {
+  flags = stringToFlags(flags);
```

Comment:
Let's move this down to be with the other flags validation happening on 1691

Reply:
I ended up moving that line, along with the validation, up a little further in the function -- `flags` gets used prior to line 1691, but until this most recent change, wasn't actually validated until that line.  I figured it would make more sense to validate them almost immediately, or at least do that before they're used.  Thank you for pointing this out, though -- I don't think I would have caught that mistake if it weren't for this suggestion!